doc/connection-to-the-internet-of-things/how-to-read-measures-captured-from-iot-devices.md

```diff
@@ -1,121 +1,241 @@
+For a more detailed description of the IDAS APIs,
+please have a look [here](http://docs.telefonicaiotiotagents.apiary.io/).
 
 Basically, there are four simple steps to follow:
 
-**1. Create an IDAS Service**  
- If you are using the public IDAS instance with the public “OpenIoT”
-testing service available at 130.206.80.47 (Port 5073) you may skip this
+1. **Create an IDAS Service**
+
+ If you are using the public IDAS instance with the public `openiot`
+testing service available at `130.206.80.47` (Port `5073`) you may skip this
 step. Just keep in mind the shared secret for this public service (that
-your devices need to know) is the string “4jggokgpepnvsb2uv4s40d59ov”.  
- Otherwise, creating an IDAS Service is basically a simple HTTP POST
-like this:
+your devices need to know) is the string `4jggokgpepnvsb2uv4s40d59ov`.  
+ Otherwise, creating an IDAS Service consists of a simple HTTP POST like this:
 
-    POST <idas_host>:<idas_port>/iot/services
-    //Example: HTTP POST: http://130.206.80.40:5371/iot/services 
-    Headers: {'content-type': 'application/json’; 'X-Auth-Token' : [TOKEN]; "Fiware-Service: OpenIoT”; "Fiware-ServicePath: /"}
+    POST http://130.206.80.40:5371/iot/services
+    
+    Headers:
+    
+    {
+      'Content-Type':       'application/json’,
+      'X-Auth-Token' :      '[TOKEN]',
+      'Fiware-Service':     'openiot',
+      'Fiware-ServicePath': '/'
+    }
+    
     Payload:
+    
     {
       "services": [
         {
-        "apikey": "4jggokgpepnvsb2uv4s40d59ov",
-        "token": "token2",
-        "cbroker": "http://0.0.0.0:1026",
-        "entity_type": "thing",
-        "resource": "/iot/d"
+          "apikey":      "4jggokgpepnvsb2uv4s40d59ov",
+          "token":       "aToken",
+          "cbroker":     "http://0.0.0.0:1026",
+          "entity_type": "thing",
+          "resource":    "/iot/d"
         }
       ]
     }
 
-Where 0.0.0.0:1026 might be replaced by a private instance of
-ContextBroker or just leave it like this (0.0.0.0:1026) if the public
-instance at the same VM (130.206.80.40:1026) will be used. The apikey
-string should be updated with a shared secret to be known by devices.
+Where `0.0.0.0:1026` might be replaced by a private instance of
+a Context Broker or just leave it as it is (`0.0.0.0:1026`) if the public
+instance is running at the same VM (`130.206.80.40:1026`) will be used. The apikey
+string should be updated with a shared secret to be known by your IoT devices. An OAuth
+token is needed as well, a simple way of obtaining one
+is described [here](http://fiware-orion.readthedocs.io/en/develop/quick_start_guide/index.html).
+
+Likewise, you may want to experiment using the *FIWARE Tour Guide Application*.
+Assuming you have installed it locally, you can follow these steps: 
+
+  POST http://localhost:4041/iot/services/
+  
+  Headers:
```

Comment:
```
  POST http://localhost:4041/iot/services/
  
  Headers:
```
Need four spaces tab instead of two:
```
    POST http://localhost:4041/iot/services/
  
    Headers:
```